packages/api/tsserver/tsservers.mts

```diff
@@ -44,8 +44,9 @@ export class TsServers {
     // created, the dependencies are not installed and thus this will
     // shut down immediately. Make sure that we handle this case after
     // package.json has finished installing its deps.
-    const child = spawn('npx', ['tsserver'], {
+    const child = spawn('pnpm', ['dlx' ,'tsserver'], {
```

Comment:
We don't want to do this, because `pnpm` is a dependency for dev, but not for users. This would require our users to have pnpm installed.

Reply:
Ooo okay, I didn't think that way. I will look into this further. Thank you!